projects/core/src/cms/config/default-cms-config.ts

```diff
@@ -4,4 +4,13 @@ export const defaultCmsModuleConfig: CmsConfig = {
   cmsComponents: {
     CMSTabParagraphComponent: { selector: 'cx-paragraph' },
   },
+  backend: {
+    occ: {
+      endpoints: {
+        component: 'cms/components/${id}',
```

Comment:
Is it possible to configure fields for the component endpoint as well? We had a question on why not all component properties are exposed, most likely due to the backend populators, but could also be influence here.

Reply:
Well in this case I didn't add a `fields` parameter because we dont use it by default. A customer could customize it to make it like: